src/components/DeploymentForm.vue

```diff
@@ -100,10 +100,15 @@
     return mapper.map('SchemaResponse', rawSchema ?? {}, 'Schema')
   })
 
+  const parameters = computed(() => {
+    const source = { values: props.deployment.parameters, schema: parameterOpenApiSchema.value }
+    return mapper.map('SchemaValuesResponse', source, 'SchemaValues')
+  })
```

Comment:
This is necessary because the parameters object is being used by the schema form input components but without the type resolution rigor default values go through. This means that values supplied through the parameters blob don't necessarily confirm to what underlying components need